.editorconfig

```diff
@@ -249,6 +249,10 @@ dotnet_diagnostic.AV1555.severity = suggestion
 dotnet_diagnostic.AV1561.severity = suggestion
 # AV1564: Parameter in public or internal member is of type bool or bool?
 dotnet_diagnostic.AV1564.severity = suggestion
+# AV1554: Do not use optional parameters in interface methods or their concrete implementations
+dotnet_diagnostic.AV1554.severity = none
+# AV1580: Argument for parameter calls nested method
+dotnet_diagnostic.AV1580.severity = none
```

Comment:
<details>
<summary>AV1554</summary>

```
Method 'ExceptionAssertions<TException>.WithMessage(string, string, params object[])' contains optional parameter 'because'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	81
Method 'ExceptionAssertions<TException>.WithInnerException<TInnerException>(string, params object[])' contains optional parameter 'because'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	107
Method 'ExceptionAssertions<TException>.WithInnerExceptionExactly<TInnerException>(string, params object[])' contains optional parameter 'because'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	142
```

</details>

<details>
<summary>AV1580</summary>

```
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'XNode.DeepEquals(XNode?, XNode?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Xml\XElementAssertions.cs	44
Argument for parameter 'line' in method call to 'FormattedObjectGraph.AddLine(string)' calls nested method 'StringExtensions.EscapePlaceholders(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Xml\XmlNodeFormatter.cs	25
Argument for parameter 'collection' in method call to 'List<T>.AddRange(IEnumerable<T>)' calls nested method 'Enumerable.OfType<T>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AggregateExceptionExtractor.cs	31
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.Equals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Xml\XDocumentAssertions.cs	44
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<TMatchedElement, string>(IEnumerable<TMatchedElement>, Func<TMatchedElement, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AndWhichConstraint.cs	43
Argument for parameter 'item' in method call to 'List<AttributeData>.Add(AttributeData)' calls nested method 'AttributeData.AttributeData(string, string, string, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Xml\Equivalency\XmlIterator.cs	67
Argument for parameter 'actualValue' in method call to 'GenericCollectionAssertions<DataTable>.GenericCollectionAssertions(IEnumerable<DataTable>)' calls nested method 'ReadOnlyNonGenericCollectionWrapper.Create(DataTableCollection)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	396
Argument for parameter 'actualValue' in method call to 'GenericCollectionAssertions<DataRow>.GenericCollectionAssertions(IEnumerable<DataRow>)' calls nested method 'ReadOnlyNonGenericCollectionWrapper.Create(DataRowCollection)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	416
Argument for parameter 'actualValue' in method call to 'GenericCollectionAssertions<DataColumn>.GenericCollectionAssertions(IEnumerable<DataColumn>)' calls nested method 'ReadOnlyNonGenericCollectionWrapper.Create(DataColumnCollection)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	406
Argument for parameter 'types' in method call to 'TypeSelectorAssertions.TypeSelectorAssertions(params Type[])' calls nested method 'TypeSelector.ToArray()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	793
Argument for parameter 'methods' in method call to 'MethodInfoSelectorAssertions.MethodInfoSelectorAssertions(params MethodInfo[])' calls nested method 'MethodInfoSelector.ToArray()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	828
Argument for parameter 'properties' in method call to 'PropertyInfoSelectorAssertions.PropertyInfoSelectorAssertions(params PropertyInfo[])' calls nested method 'PropertyInfoSelector.ToArray()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\AssertionExtensions.cs	853
Argument for parameter 'expected' in method call to 'GenericCollectionAssertions<TCollection, string, TAssertions>.Equal(IEnumerable<string>, string, params object[])' calls nested method 'Enumerable.AsEnumerable<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\StringCollectionAssertions.cs	55
Argument for parameter 'obj' in method call to 'Action<string>.Invoke(string)' calls nested method 'StackFrame.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\CallerIdentifier.cs	53
Argument for parameter 'obj' in method call to 'Action<string>.Invoke(string)' calls nested method 'Exception.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\CallerIdentifier.cs	68
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<string>, EquivalencyAssertionOptions<string>>.Invoke(EquivalencyAssertionOptions<string>)' calls nested method 'AssertionOptions.CloneDefaults<string>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\StringCollectionAssertions.cs	124
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<IEnumerable<string>>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\StringCollectionAssertions.cs	126
Argument for parameter 'message' in method call to 'AssertionScope.FailWith(string)' calls nested method 'TypeAssertions.GetFailureMessageIfTypesAreDifferent(Type, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	58
Argument for parameter 'item' in method call to 'List<Type>.Add(Type)' calls nested method 'Enumerable.Single<Type>(IEnumerable<Type>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeSelector.cs	246
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TAttribute>(IEnumerable<TAttribute>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	236
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TAttribute>(IEnumerable<TAttribute>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	295
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TAttribute>(IEnumerable<TAttribute>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	269
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TAttribute>(IEnumerable<TAttribute>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	328
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TypeExtensions.IsCSharpSealed(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	668
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TypeExtensions.IsCSharpAbstract(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	732
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TypeExtensions.IsCSharpStatic(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	796
Argument for parameter 'second' in method call to 'Enumerable.Concat<MemberInfo>(IEnumerable<MemberInfo>, IEnumerable<MemberInfo>)' calls nested method 'TypeExtensions.GetNonPrivateFields(Type, MemberVisibility)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\TypeExtensions.cs	214
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.ReferenceEquals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\DataColumnCollectionAssertionExtensions.cs	39
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.ReferenceEquals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\DataRowCollectionAssertionExtensions.cs	37
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<Type, string>(IEnumerable<Type>, Func<Type, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\TypeAssertions.cs	1503
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	92
Argument for parameter 'arg1' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue(TCollection, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	92
Argument for parameter 'arg2' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue<T>(T, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	92
Argument for parameter 'arg1' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue(TCollection, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	142
Argument for parameter 'arg2' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue<T>(T, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	142
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	212
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<TExpectation>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	214
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'StringCollectionAssertions<TCollection, TAssertions>.ContainsMatch(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\StringCollectionAssertions.cs	255
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'StringCollectionAssertions<TCollection, TAssertions>.NotContainsMatch(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\StringCollectionAssertions.cs	338
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.All<T>(IEnumerable<T>, Func<T, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	79
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enumerable.All<T>(IEnumerable<T>, Func<T, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	82
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'GenericCollectionAssertions<TCollection, T, TAssertions>.GetType<T>(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	82
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<T, string>(IEnumerable<T>, Func<T, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	83
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'GenericCollectionAssertions<TCollection, T, TAssertions>.GetType<T>(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	119
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<T, string>(IEnumerable<T>, Func<T, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	120
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.ReferenceEquals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\DataTableCollectionAssertionExtensions.cs	34
Argument for parameter 'stream' in method call to 'StreamReader.StreamReader(Stream)' calls nested method 'File.OpenRead(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\CallerIdentifier.cs	197
Argument for parameter 'value' in method call to 'Dictionary<Predicate<TValue>, MaximumMatchingSolver<TValue>.Match>.Add(Predicate<TValue>, MaximumMatchingSolver<TValue>.Match)' calls nested method 'MaximumMatchingSolver<TValue>.Match.Match()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\MaximumMatching\MaximumMatchingSolver.cs	151
Argument for parameter 'startIndex' in method call to 'string.Substring(int)' calls nested method 'Math.Min(int, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\CallerIdentifier.cs	223
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.All<T>(IEnumerable<T>, Func<T, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	223
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enumerable.All<T>(IEnumerable<T>, Func<T, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	226
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<T, string>(IEnumerable<T>, Func<T, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	227
Argument for parameter 'collection' in method call to 'List<Predicate<TValue>>.AddRange(IEnumerable<Predicate<TValue>>)' calls nested method 'Enumerable.Select<Expression<Func<TValue, bool>>, Predicate<TValue>>(IEnumerable<Expression<Func<TValue, bool>>>, Func<Expression<Func<TValue, bool>>, int, Predicate<TValue>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\MaximumMatching\MaximumMatchingProblem.cs	22
Argument for parameter 'collection' in method call to 'List<Element<TValue>>.AddRange(IEnumerable<Element<TValue>>)' calls nested method 'Enumerable.Select<TValue, Element<TValue>>(IEnumerable<TValue>, Func<TValue, int, Element<TValue>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\MaximumMatching\MaximumMatchingProblem.cs	23
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'DataColumnCollection.Contains(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataTableAssertions.cs	124
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<T, string>(IEnumerable<T>, Func<T, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	264
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	354
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<IEnumerable<TExpectation>>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	356
Argument for parameter 'comparer' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.BeInOrder(IComparer<T>, SortOrder, string, params object[])' calls nested method 'Comparer<T>.Create(Comparison<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	486
Argument for parameter 'arg' in method call to 'Func<IDataEquivalencyAssertionOptions<DataTable>, IDataEquivalencyAssertionOptions<DataTable>>.Invoke(IDataEquivalencyAssertionOptions<DataTable>)' calls nested method 'AssertionOptions.CloneDefaults<DataTable, DataEquivalencyAssertionOptions<DataTable>>(Func<EquivalencyAssertionOptions, DataEquivalencyAssertionOptions<DataTable>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataTableAssertions.cs	246
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<DataTable>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataTableAssertions.cs	248
Argument for parameter 'comparer' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.BeInOrder(IComparer<T>, SortOrder, string, params object[])' calls nested method 'Comparer<T>.Create(Comparison<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	601
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'DataTableCollection.Contains(string?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataSetAssertions.cs	124
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'ICollection<T>.Contains(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	685
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'DataColumnCollection.Contains(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataRowAssertions.cs	94
Argument for parameter 'arg' in method call to 'Func<IDataEquivalencyAssertionOptions<DataSet>, IDataEquivalencyAssertionOptions<DataSet>>.Invoke(IDataEquivalencyAssertionOptions<DataSet>)' calls nested method 'AssertionOptions.CloneDefaults<DataSet, DataEquivalencyAssertionOptions<DataSet>>(Func<EquivalencyAssertionOptions, DataEquivalencyAssertionOptions<DataSet>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataSetAssertions.cs	233
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<DataSet>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataSetAssertions.cs	242
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<T>(IEnumerable<T>, Func<T, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	722
Argument for parameter 'arg' in method call to 'Func<IDataEquivalencyAssertionOptions<DataRow>, IDataEquivalencyAssertionOptions<DataRow>>.Invoke(IDataEquivalencyAssertionOptions<DataRow>)' calls nested method 'AssertionOptions.CloneDefaults<DataRow, DataEquivalencyAssertionOptions<DataRow>>(Func<EquivalencyAssertionOptions, DataEquivalencyAssertionOptions<DataRow>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataRowAssertions.cs	181
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<DataRow>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataRowAssertions.cs	183
Argument for parameter 'referencedMember' in method call to 'DataEquivalencyAssertionOptions<T>.BuildMemberSelectionPredicate(Type, MemberInfo)' calls nested method 'DataEquivalencyAssertionOptions<T>.GetMemberAccessTargetMember(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	159
Argument for parameter 'referencedMember' in method call to 'DataEquivalencyAssertionOptions<T>.BuildMemberSelectionPredicate(Type, MemberInfo)' calls nested method 'DataEquivalencyAssertionOptions<T>.GetMemberAccessTargetMember(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	169
Argument for parameter 'trace' in method call to 'ITraceWriter.AddSingle(string)' calls nested method 'GetTraceMessage.Invoke(INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Tracing\Tracer.cs	28
Argument for parameter 'trace' in method call to 'ITraceWriter.AddBlock(string)' calls nested method 'GetTraceMessage.Invoke(INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Tracing\Tracer.cs	42
Argument for parameter 'left' in method call to 'Expression.Equal(Expression, Expression)' calls nested method 'Expression.MakeMemberAccess(Expression?, MemberInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	197
Argument for parameter 'member' in method call to 'Expression.MakeMemberAccess(Expression?, MemberInfo)' calls nested method 'Type.GetProperty(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	199
Argument for parameter 'right' in method call to 'Expression.Equal(Expression, Expression)' calls nested method 'Expression.Constant(object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	200
Argument for parameter 'left' in method call to 'Expression.Equal(Expression, Expression)' calls nested method 'Expression.MakeMemberAccess(Expression?, MemberInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	203
Argument for parameter 'member' in method call to 'Expression.MakeMemberAccess(Expression?, MemberInfo)' calls nested method 'Type.GetProperty(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	205
Argument for parameter 'value' in method call to 'StringBuilder.Append(string?)' calls nested method 'string.String(char, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Tracing\StringBuilderTraceWriter.cs	33
Argument for parameter 'right' in method call to 'Expression.Equal(Expression, Expression)' calls nested method 'Expression.Constant(object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataEquivalencyAssertionOptions.cs	206
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	844
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<TExpectation>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	852
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\StructuralEqualityEquivalencyStep.cs	74
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\StructuralEqualityEquivalencyStep.cs	103
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<object>(IEnumerable<object>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\EventRaisingExtensions.cs	30
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<OccurredEvent>(IEnumerable<OccurredEvent>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\EventRaisingExtensions.cs	49
Argument for parameter 'parameterTypes' in method call to 'TypeExtensions.GetMethod(Type, string, IEnumerable<Type>)' calls nested method 'Enumerable.Empty<Type>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\TypeExtensions.cs	391
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<Expression<Func<T, bool>>, string>(IEnumerable<Expression<Func<T, bool>>>, Func<Expression<Func<T, bool>>, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\EventRaisingExtensions.cs	139
Argument for parameter 'expectation' in method call to 'EnumerableEquivalencyValidator.Execute<T>(object[], T[])' calls nested method 'GenericEnumerableEquivalencyStep.ToArray<T>(IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericEnumerableEquivalencyStep.cs	60
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'GenericEnumerableEquivalencyStep.IsCollection(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericEnumerableEquivalencyStep.cs	71
Argument for parameter 'type' in method call to 'GenericEnumerableEquivalencyStep.IsCollection(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericEnumerableEquivalencyStep.cs	71
Argument for parameter 'matchedConstraint' in method call to 'AndWhichConstraint<TAssertions, IEnumerable<TValue>>.AndWhichConstraint(TAssertions, IEnumerable<TValue>)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.RepetitionPreservingIntersect(IEnumerable<TValue>, IEnumerable<TValue>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	518
Argument for parameter 'second' in method call to 'Enumerable.Intersect<TValue>(IEnumerable<TValue>, IEnumerable<TValue>)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValues(TCollection)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	609
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	776
Argument for parameter 'arg1' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue(TCollection, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	702
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'GenericCollectionAssertions<TCollection, T, TAssertions>.GetType<T>(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1043
Argument for parameter 'arg1' in method call to 'Func<TValue, TValue, bool>.Invoke(TValue, TValue)' calls nested method 'GenericDictionaryAssertions<TCollection, TKey, TValue, TAssertions>.GetValue(TCollection, TKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericDictionaryAssertions.cs	845
Argument for parameter 'expected' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.AssertCollectionEndsWith<T, TExpectation>(IEnumerable<T>, ICollection<TExpectation>, Func<T, TExpectation, bool>, string, params object[])' calls nested method 'EnumerableExtensions.ConvertOrCastToCollection<TExpectation>(IEnumerable<TExpectation>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1192
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<T>(IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1125
Argument for parameter 'predicate' in method call to 'Enumerable.Where<T>(IEnumerable<T>, Func<T, bool>)' calls nested method 'Expression<Func<T, bool>>.Compile()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1129
Argument for parameter 'equalityComparison' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.EndWith<T>(IEnumerable<T>, Func<T, T, bool>, string, params object[])' calls nested method 'ObjectExtensions.GetComparer<T>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1212
Argument for parameter 'equalityComparison' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.AssertSubjectEquality<T>(IEnumerable<T>, Func<T, T, bool>, string, params object[])' calls nested method 'ObjectExtensions.GetComparer<T>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1222
Argument for parameter 'equalityComparison' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.AssertSubjectEquality<T>(IEnumerable<T>, Func<T, T, bool>, string, params object[])' calls nested method 'ObjectExtensions.GetComparer<T>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1266
Argument for parameter 'item' in method call to 'BlockingCollection<RecordedEvent>.Add(RecordedEvent)' calls nested method 'RecordedEvent.RecordedEvent(DateTime, int, params object[])'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventRecorder.cs	84
Argument for parameter 'utcNow' in method call to 'RecordedEvent.RecordedEvent(DateTime, int, params object[])' calls nested method 'Func<DateTime>.Invoke()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventRecorder.cs	84
Argument for parameter 'sequence' in method call to 'RecordedEvent.RecordedEvent(DateTime, int, params object[])' calls nested method 'ThreadSafeSequenceGenerator.Increment()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventRecorder.cs	84
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<T, T, bool>.Invoke(T, T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1496
Argument for parameter 'second' in method call to 'Enumerable.Concat<Type>(IEnumerable<Type>, IEnumerable<Type>)' calls nested method 'Type.GetInterfaces()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventMonitor.cs	110
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<T>(IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1644
Argument for parameter 'unexpected' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.NotBeEquivalentTo<TExpectation>(IEnumerable<TExpectation>, Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>, string, params object[])' calls nested method 'EnumerableExtensions.ConvertOrCastToList<TExpectation>(IEnumerable<TExpectation>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1713
Argument for parameter 'comparer' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.NotBeInOrder(IComparer<T>, SortOrder, string, params object[])' calls nested method 'Comparer<T>.Create(Comparison<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1876
Argument for parameter 'comparer' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.NotBeInOrder(IComparer<T>, SortOrder, string, params object[])' calls nested method 'Comparer<T>.Create(Comparison<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	1990
Argument for parameter 'parameterTypes' in method call to 'DynamicMethod.DynamicMethod(string, Type?, Type[]?, Module)' calls nested method 'EventHandlerFactory.AppendParameterListThisReference(Type[])'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventHandlerFactory.cs	27
Argument for parameter 'item' in method call to 'List<ConversionSelector.ConversionSelectorRule>.Add(ConversionSelector.ConversionSelectorRule)' calls nested method 'ConversionSelector.ConversionSelectorRule.ConversionSelectorRule(Func<IObjectInfo, bool>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	46
Argument for parameter 'item' in method call to 'List<ConversionSelector.ConversionSelectorRule>.Add(ConversionSelector.ConversionSelectorRule)' calls nested method 'ConversionSelector.ConversionSelectorRule.ConversionSelectorRule(Func<IObjectInfo, bool>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	53
Argument for parameter 'predicate' in method call to 'ConversionSelector.ConversionSelectorRule.ConversionSelectorRule(Func<IObjectInfo, bool>, string)' calls nested method 'Expression<Func<IObjectInfo, bool>>.Compile()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	54
Argument for parameter 'item' in method call to 'List<ConversionSelector.ConversionSelectorRule>.Add(ConversionSelector.ConversionSelectorRule)' calls nested method 'ConversionSelector.ConversionSelectorRule.ConversionSelectorRule(Func<IObjectInfo, bool>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	62
Argument for parameter 'predicate' in method call to 'ConversionSelector.ConversionSelectorRule.ConversionSelectorRule(Func<IObjectInfo, bool>, string)' calls nested method 'Expression<Func<IObjectInfo, bool>>.Compile()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	63
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<string>(IEnumerable<string>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Events\EventAssertions.cs	113
Argument for parameter 'inclusions' in method call to 'ConversionSelector.ConversionSelector(List<ConversionSelector.ConversionSelectorRule>, List<ConversionSelector.ConversionSelectorRule>)' calls nested method 'List<ConversionSelector.ConversionSelectorRule>.List(IEnumerable<ConversionSelector.ConversionSelectorRule>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	103
Argument for parameter 'exclusions' in method call to 'ConversionSelector.ConversionSelector(List<ConversionSelector.ConversionSelectorRule>, List<ConversionSelector.ConversionSelectorRule>)' calls nested method 'List<ConversionSelector.ConversionSelectorRule>.List(IEnumerable<ConversionSelector.ConversionSelectorRule>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\ConversionSelector.cs	103
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	2237
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<TExpectation>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	2245
Argument for parameter 'arg' in method call to 'Func<IDataEquivalencyAssertionOptions<DataColumn>, IDataEquivalencyAssertionOptions<DataColumn>>.Invoke(IDataEquivalencyAssertionOptions<DataColumn>)' calls nested method 'AssertionOptions.CloneDefaults<DataColumn, DataEquivalencyAssertionOptions<DataColumn>>(Func<EquivalencyAssertionOptions, DataEquivalencyAssertionOptions<DataColumn>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataColumnAssertions.cs	116
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<DataColumn>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Data\DataColumnAssertions.cs	118
Argument for parameter 'selectionRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<EquivalencyAssertionOptions<TExpectation>>.AddSelectionRule(IMemberSelectionRule)' calls nested method 'ExcludeMemberByPathSelectionRule.ExcludeMemberByPathSelectionRule(MemberPath)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	38
Argument for parameter 'pathToExclude' in method call to 'ExcludeMemberByPathSelectionRule.ExcludeMemberByPathSelectionRule(MemberPath)' calls nested method 'ExpressionExtensions.GetMemberPath<TExpectation, object>(Expression<Func<TExpectation, object>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	38
Argument for parameter 'pathToExclude' in method call to 'ExcludeMemberByPathSelectionRule.ExcludeMemberByPathSelectionRule(MemberPath)' calls nested method 'ExpressionExtensions.GetMemberPath<TExpectation, IEnumerable<TNext>>(Expression<Func<TExpectation, IEnumerable<TNext>>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	48
Argument for parameter 'selectionRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<EquivalencyAssertionOptions<TExpectation>>.AddSelectionRule(IMemberSelectionRule)' calls nested method 'IncludeMemberByPathSelectionRule.IncludeMemberByPathSelectionRule(MemberPath)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	61
Argument for parameter 'pathToInclude' in method call to 'IncludeMemberByPathSelectionRule.IncludeMemberByPathSelectionRule(MemberPath)' calls nested method 'ExpressionExtensions.GetMemberPath<TExpectation, object>(Expression<Func<TExpectation, object>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	61
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'PathBasedOrderingRule.PathBasedOrderingRule(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	73
Argument for parameter 'defaults' in method call to 'EquivalencyAssertionOptions<IEnumerable<TExpectation>>.EquivalencyAssertionOptions(IEquivalencyAssertionOptions)' calls nested method 'CollectionMemberAssertionOptionsDecorator.CollectionMemberAssertionOptionsDecorator(IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	83
Argument for parameter 'matchingRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<EquivalencyAssertionOptions<TExpectation>>.AddMatchingRule(IMemberMatchingRule)' calls nested method 'MappedPathMatchingRule.MappedPathMatchingRule(string, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	125
Argument for parameter 'expectationMemberPath' in method call to 'EquivalencyAssertionOptions<TExpectation>.WithMapping(string, string)' calls nested method 'StringExtensions.WithoutSpecificCollectionIndices(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	102
Argument for parameter 'subjectMemberPath' in method call to 'EquivalencyAssertionOptions<TExpectation>.WithMapping(string, string)' calls nested method 'StringExtensions.WithoutSpecificCollectionIndices(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	103
Argument for parameter 'matchingRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<EquivalencyAssertionOptions<TExpectation>>.AddMatchingRule(IMemberMatchingRule)' calls nested method 'MappedMemberMatchingRule<TNestedExpectation, TNestedSubject>.MappedMemberMatchingRule(string, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	168
Argument for parameter 'expectationMemberName' in method call to 'EquivalencyAssertionOptions<TExpectation>.WithMapping<TNestedExpectation, TNestedSubject>(string, string)' calls nested method 'MemberPath.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	147
Argument for parameter 'subjectMemberName' in method call to 'EquivalencyAssertionOptions<TExpectation>.WithMapping<TNestedExpectation, TNestedSubject>(string, string)' calls nested method 'MemberPath.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyAssertionOptions.cs	148
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'Comparands.GetExpectedType(IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyStep.cs	10
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<string, string>(IEnumerable<string>, Func<string, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	2857
Argument for parameter 'equalityComparison' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.StartWith<T>(IEnumerable<T>, Func<T, T, bool>, string, params object[])' calls nested method 'ObjectExtensions.GetComparer<T>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3114
Argument for parameter 'expected' in method call to 'GenericCollectionAssertions<TCollection, T, TAssertions>.AssertCollectionStartsWith<T, TExpectation>(IEnumerable<T>, ICollection<TExpectation>, Func<T, TExpectation, bool>, string, params object[])' calls nested method 'EnumerableExtensions.ConvertOrCastToCollection<TExpectation>(IEnumerable<TExpectation>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3094
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.FromCollectionItem<TItem>(string, INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyValidationContext.cs	44
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.SequenceEqual<T>(IEnumerable<T>, IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3137
Argument for parameter 'parentConstraint' in method call to 'AndConstraint<SubsequentOrderingAssertions<T>>.AndConstraint(SubsequentOrderingAssertions<T>)' calls nested method 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3143
Argument for parameter 'parentConstraint' in method call to 'AndConstraint<SubsequentOrderingAssertions<T>>.AndConstraint(SubsequentOrderingAssertions<T>)' calls nested method 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3147
Argument for parameter 'previousOrderedEnumerable' in method call to 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)' calls nested method 'Enumerable.OrderBy<T, T>(IEnumerable<T>, Func<T, T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3147
Argument for parameter 'assemblyRef' in method call to 'Assembly.Load(AssemblyName)' calls nested method 'AssemblyName.AssemblyName(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\XUnit2TestFramework.cs	18
Argument for parameter 'tracingBlock' in method call to 'AssertionScope.AppendTracing(string)' calls nested method 'ITraceWriter.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyValidator.cs	26
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<string, string>(IEnumerable<string>, Func<string, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	2945
Argument for parameter 'objA' in method call to 'object.ReferenceEquals(object?, object?)' calls nested method 'Enumerable.First<T>(IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3262
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<Predicate<T>, string>(IEnumerable<Predicate<T>>, Func<Predicate<T>, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3028
Argument for parameter 'objA' in method call to 'object.ReferenceEquals(object?, object?)' calls nested method 'Enumerable.Last<T>(IEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3267
Argument for parameter 'getTraceMessage' in method call to 'Tracer.WriteLine(GetTraceMessage)' calls nested method 'Func<IEquivalencyStep, GetTraceMessage>.Invoke(IEquivalencyStep)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyValidator.cs	74
Argument for parameter 'orderingRules' in method call to 'OrderingRuleCollection.OrderingRuleCollection(IEnumerable<IOrderingRule>)' calls nested method 'Enumerable.Select<IOrderingRule, CollectionMemberOrderingRuleDecorator>(IEnumerable<IOrderingRule>, Func<IOrderingRule, CollectionMemberOrderingRuleDecorator>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Execution\CollectionMemberAssertionOptionsDecorator.cs	39
Argument for parameter 'second' in method call to 'Enumerable.SequenceEqual<string>(IEnumerable<string>, IEnumerable<string>)' calls nested method 'Enumerable.Take<string>(IEnumerable<string>, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Execution\ObjectReference.cs	52
Argument for parameter 'replacement' in method call to 'Regex.Replace(string, string, string)' calls nested method 'MessageBuilder.SanitizeReason(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\MessageBuilder.cs	35
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<string, string>(IEnumerable<string>, Func<string, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3311
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'XAttribute.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\XAttributeValueFormatter.cs	21
Argument for parameter 'line' in method call to 'FormattedObjectGraph.AddLine(string)' calls nested method 'FormattableString.Invariant(FormattableString)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\AggregateExceptionValueFormatter.cs	31
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'TimeOnly.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\TimeOnlyValueFormatter.cs	25
Argument for parameter 'value' in method call to 'FormatChild.Invoke(string, object, FormattedObjectGraph)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\TaskFormatter.cs	20
Argument for parameter 'parentConstraint' in method call to 'AndConstraint<SubsequentOrderingAssertions<T>>.AndConstraint(SubsequentOrderingAssertions<T>)' calls nested method 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3407
Argument for parameter 'previousOrderedEnumerable' in method call to 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)' calls nested method 'Enumerable.OrderBy<T, T>(IEnumerable<T>, Func<T, T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3407
Argument for parameter 'parentConstraint' in method call to 'AndConstraint<SubsequentOrderingAssertions<T>>.AndConstraint(SubsequentOrderingAssertions<T>)' calls nested method 'SubsequentOrderingAssertions<T>.SubsequentOrderingAssertions(IEnumerable<T>, IOrderedEnumerable<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Collections\GenericCollectionAssertions.cs	3412
Argument for parameter 'index' in method call to 'GivenSelectorExtensions.CollectionWithIndex<TActual>.CollectionWithIndex(ICollection<TActual>, int)' calls nested method 'Func<ICollection<TActual>, ICollection<TExpected>, int>.Invoke(ICollection<TActual>, ICollection<TExpected>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\GivenSelectorExtensions.cs	67
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'float.ToString(IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\SingleValueFormatter.cs	34
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<T, bool>.Invoke(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\GivenSelector.cs	42
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'string.Join(string?, IEnumerable<string?>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\PredicateLambdaExpressionValueFormatter.cs	24
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<Expression, string>(IEnumerable<Expression>, Func<Expression, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\PredicateLambdaExpressionValueFormatter.cs	24
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'Expression.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\PredicateLambdaExpressionValueFormatter.cs	28
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.FromDictionaryItem<TExpectation>(object, INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyValidationContext.cs	54
Argument for parameter 'assertionStrategy' in method call to 'AssertionScope.AssertionScope(IAssertionStrategy, AssertionScope)' calls nested method 'DefaultAssertionStrategy.DefaultAssertionStrategy()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	129
Argument for parameter 'type' in method call to 'IEquivalencyAssertionOptions.GetEqualityStrategy(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\EquivalencyValidationContext.cs	74
Argument for parameter 'dateTime' in method call to 'DateTimeOffset.DateTimeOffset(DateTime, TimeSpan)' calls nested method 'DateTime.SpecifyKind(DateTime, DateTimeKind)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\DateTimeExtensions.cs	18
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'string.Join(string?, IEnumerable<string?>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\CollectingAssertionStrategy.cs	38
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	199
Argument for parameter 'obj' in method call to 'Action<string>.Invoke(string)' calls nested method 'StringBuilder.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\CollectingAssertionStrategy.cs	48
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	200
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.Clone()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\ContextDataItems.cs	39
Argument for parameter 'message' in method call to 'IAssertionStrategy.HandleFailure(string)' calls nested method 'StringExtensions.Capitalize(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	274
Argument for parameter 'args' in method call to 'FailReason.FailReason(string, params object[])' calls nested method 'Enumerable.ToArray<object>(IEnumerable<object>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	303
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	343
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	352
Argument for parameter 'item' in method call to 'ContextDataItems.Add(ContextDataItems.DataItem)' calls nested method 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	361
Argument for parameter 'value' in method call to 'ContextDataItems.DataItem.DataItem(string, object, bool, bool)' calls nested method 'AssertionScope.DeferredReportable.DeferredReportable(Func<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	361
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateOnly.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateOnlyValueFormatter.cs	25
Argument for parameter 'tracingBlock' in method call to 'AssertionScope.AppendTracing(string)' calls nested method 'StringBuilder.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	398
Argument for parameter 'value' in method call to 'IDictionary<string, object>.Add(string, object)' calls nested method 'StringBuilder.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Execution\AssertionScope.cs	408
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragmentOnNewLine(string)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DefaultValueFormatter.cs	45
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DefaultValueFormatter.cs	49
Argument for parameter 'line' in method call to 'FormattedObjectGraph.AddLine(string)' calls nested method 'DefaultValueFormatter.TypeDisplayName(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DefaultValueFormatter.cs	75
Argument for parameter 'enumerable' in method call to 'Iterator<MemberInfo>.Iterator(IEnumerable<MemberInfo>, int)' calls nested method 'Enumerable.OrderBy<MemberInfo, string>(IEnumerable<MemberInfo>, Func<MemberInfo, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DefaultValueFormatter.cs	79
Argument for parameter 'value' in method call to 'Expression.Constant(object?)' calls nested method 'Delegate.DynamicInvoke(params object?[]?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\PredicateLambdaExpressionValueFormatter.cs	96
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateTimeOffset.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateTimeOffsetValueFormatter.cs	41
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateTimeOffset.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateTimeOffsetValueFormatter.cs	54
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateTimeOffset.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateTimeOffsetValueFormatter.cs	58
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateTimeOffset.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateTimeOffsetValueFormatter.cs	62
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DateTimeOffset.ToString(string?, IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DateTimeOffsetValueFormatter.cs	66
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'int.ToString(IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\Int32ValueFormatter.cs	21
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'DoubleValueFormatter.Format(object)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\DoubleValueFormatter.cs	22
Argument for parameter 'childPath' in method call to 'FormatChild.Invoke(string, object, FormattedObjectGraph)' calls nested method 'string.Join<int>(string?, IEnumerable<int>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\MultidimensionalArrayFormatter.cs	49
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Reverse<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\Formatter.cs	211
Argument for parameter 'value' in method call to 'string.Join(string?, params string?[])' calls nested method 'Enumerable.ToArray<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\Formatter.cs	222
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Concat<string>(IEnumerable<string>, IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\FormattedObjectGraph.cs	126
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'string.Replace(string, string?, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\ExpressionValueFormatter.cs	22
Argument for parameter 'fragment' in method call to 'FormattedObjectGraph.AddFragment(string)' calls nested method 'FormattableString.Invariant(FormattableString)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\ExceptionValueFormatter.cs	24
Argument for parameter 'childPath' in method call to 'FormatChild.Invoke(string, object, FormattedObjectGraph)' calls nested method 'int.ToString(IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Formatting\EnumerableValueFormatter.cs	45
Argument for parameter 'message' in method call to 'ArgumentException.ArgumentException(string?, string?)' calls nested method 'ExpressionExtensions.GetUnsupportedExpressionMessage(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ExpressionExtensions.cs	84
Argument for parameter 'message' in method call to 'ArgumentException.ArgumentException(string?, string?)' calls nested method 'ExpressionExtensions.GetUnsupportedExpressionMessage(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ExpressionExtensions.cs	93
Argument for parameter 'message' in method call to 'ArgumentException.ArgumentException(string?, string?)' calls nested method 'ExpressionExtensions.GetUnsupportedExpressionMessage(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ExpressionExtensions.cs	152
Argument for parameter 'message' in method call to 'ArgumentException.ArgumentException(string?, string?)' calls nested method 'ExpressionExtensions.GetUnsupportedExpressionMessage(Expression)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ExpressionExtensions.cs	159
Argument for parameter 'dottedPath' in method call to 'MemberPath.MemberPath(Type, Type, string)' calls nested method 'string.Replace(string, string?, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ExpressionExtensions.cs	103
Argument for parameter 'type' in method call to 'ObjectAssertionsExtensions.SimpleBinder.SimpleBinder(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\ObjectAssertionsExtensions.cs	147
Argument for parameter 'type' in method call to 'DataContractSerializer.DataContractSerializer(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\ObjectAssertionsExtensions.cs	182
Argument for parameter 'type' in method call to 'XmlSerializer.XmlSerializer(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\ObjectAssertionsExtensions.cs	226
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<T, bool>.Invoke(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Numeric\NumericAssertions.cs	468
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<T?, bool>.Invoke(T?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Numeric\NullableNumericAssertions.cs	119
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.Equals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Numeric\ComparableTypeAssertions.cs	53
Argument for parameter 'target' in method call to 'DictionaryInterfaceInfo.TryGetFromWithKey(Type, string, Type, out DictionaryInterfaceInfo)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericDictionaryEquivalencyStep.cs	76
Argument for parameter 'target' in method call to 'DictionaryInterfaceInfo.TryGetFrom(Type, string, out DictionaryInterfaceInfo)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericDictionaryEquivalencyStep.cs	84
Argument for parameter 'index' in method call to 'IEquivalencyValidationContext.AsCollectionItem<TItem>(string)' calls nested method 'int.ToString(IFormatProvider?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EquivalencyValidationContextExtensions.cs	9
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'PropertyInfoExtensions.IsVirtual(PropertyInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\PropertyInfoAssertions.cs	42
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'EquivalencyValidationContext.AsDictionaryItem<TExpectedKey, TExpectedValue>(TExpectedKey)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\GenericDictionaryEquivalencyStep.cs	206
Argument for parameter 'collection' in method call to 'List<int>.AddRange(IEnumerable<int>)' calls nested method 'Enumerable.Range(int, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	77
Argument for parameter 'objectInfo' in method call to 'OrderingRuleCollection.IsOrderingStrictFor(IObjectInfo)' calls nested method 'ObjectInfo.ObjectInfo(Comparands, INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	79
Argument for parameter 'comparands' in method call to 'ObjectInfo.ObjectInfo(Comparands, INode)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	79
Argument for parameter 'length' in method call to 'Digit.Digit(int, Digit)' calls nested method 'Array.GetLength(int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\MultiDimensionalArrayEquivalencyStep.cs	52
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\MultiDimensionalArrayEquivalencyStep.cs	39
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	181
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'EquivalencyValidationContextExtensions.AsCollectionItem<T>(IEquivalencyValidationContext, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	181
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyValidator.cs	192
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'EnumerableEquivalencyStep.IsCollection(Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyStep.cs	40
Argument for parameter 'type' in method call to 'EnumerableEquivalencyStep.IsCollection(Type)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyStep.cs	40
Argument for parameter 'type' in method call to 'EnumerableEquivalencyStep.IsCollection(Type)' calls nested method 'Comparands.GetExpectedType(IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyStep.cs	12
Argument for parameter 'subject' in method call to 'EnumerableEquivalencyValidator.Execute<object>(object[], object[])' calls nested method 'EnumerableEquivalencyStep.ToArray(object)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyStep.cs	25
Argument for parameter 'expectation' in method call to 'EnumerableEquivalencyValidator.Execute<object>(object[], object[])' calls nested method 'EnumerableEquivalencyStep.ToArray(object)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\EnumerableEquivalencyStep.cs	25
Argument for parameter 'value' in method call to 'Formatter.ToString(object, FormattingOptions)' calls nested method 'string.Substring(int, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\StringExtensions.cs	40
Argument for parameter 'objectInfo' in method call to 'IOrderingRule.Evaluate(IObjectInfo)' calls nested method 'CollectionMemberObjectInfo.CollectionMemberObjectInfo(IObjectInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Ordering\CollectionMemberOrderingRuleDecorator.cs	14
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<string, string>(IEnumerable<string>, Func<string, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\StringExtensions.cs	121
Argument for parameter 'configurationStore' in method call to 'ConfigurationStoreExceptionInterceptor.ConfigurationStoreExceptionInterceptor(IConfigurationStore)' calls nested method 'AppSettingsConfigurationStore.AppSettingsConfigurationStore()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\Services.cs	45
Argument for parameter 'second' in method call to 'Enumerable.SequenceEqual<string>(IEnumerable<string>, IEnumerable<string>, IEqualityComparer<string>?)' calls nested method 'Enumerable.Take<string>(IEnumerable<string>, int)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\MemberPath.cs	77
Argument for parameter 'second' in method call to 'Enumerable.SequenceEqual<string>(IEnumerable<string>, IEnumerable<string>, IEqualityComparer<string>?)' calls nested method 'MemberPath.GetParentSegments()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\MemberPath.cs	98
Argument for parameter 'task2' in method call to 'Task.WhenAny(Task, Task)' calls nested method 'IClock.DelayAsync(TimeSpan, CancellationToken)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\TaskCompletionSourceAssertions.cs	197
Argument for parameter 'obj' in method call to 'object.Equals(object?)' calls nested method 'ObjectExtensions.ConvertTo(object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\ObjectExtensions.cs	49
Argument for parameter 'validValues' in method call to 'TimeOnlyAssertions<TAssertions>.BeOneOf(IEnumerable<TimeOnly?>, string, params object[])' calls nested method 'Enumerable.Cast<TimeOnly?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\TimeOnlyAssertions.cs	523
Argument for parameter 'validValues' in method call to 'TimeOnlyAssertions<TAssertions>.BeOneOf(IEnumerable<TimeOnly?>, string, params object[])' calls nested method 'Enumerable.Cast<TimeOnly?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\TimeOnlyAssertions.cs	541
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<TimeOnly?>(IEnumerable<TimeOnly?>, TimeOnly?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\TimeOnlyAssertions.cs	560
Argument for parameter 'wildcardExpression' in method call to 'StringWildcardMatchingValidator.ConvertWildcardToRegEx(string)' calls nested method 'StringWildcardMatchingValidator.CleanNewLines(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringWildcardMatchingValidator.cs	36
Argument for parameter 'predicate' in method call to 'Enumerable.Where<TAttribute>(IEnumerable<TAttribute>, Func<TAttribute, bool>)' calls nested method 'Expression<Func<TAttribute, bool>>.Compile()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Common\MethodInfoExtensions.cs	40
Argument for parameter 'arg' in method call to 'Func<IMemberInfo, bool>.Invoke(IMemberInfo)' calls nested method 'MemberToMemberInfoAdapter.MemberToMemberInfoAdapter(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\IncludeMemberByPredicateSelectionRule.cs	34
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<string>(IEnumerable<string>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	89
Argument for parameter 'dottedPath' in method call to 'MemberPath.MemberPath(Type, Type, string)' calls nested method 'StringExtensions.Combine(string, string, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\IncludeMemberByPathSelectionRule.cs	26
Argument for parameter 'item' in method call to 'List<IMember>.Add(IMember)' calls nested method 'MemberFactory.Create(MemberInfo, INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\IncludeMemberByPathSelectionRule.cs	29
Argument for parameter 'arg' in method call to 'Func<IMemberInfo, bool>.Invoke(IMemberInfo)' calls nested method 'MemberToMemberInfoAdapter.MemberToMemberInfoAdapter(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\ExcludeMemberByPredicateSelectionRule.cs	27
Argument for parameter 'candidate' in method call to 'MemberPath.IsSameAs(MemberPath)' calls nested method 'MemberPath.MemberPath(IMember, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\ExcludeMemberByPathSelectionRule.cs	23
Argument for parameter 'path' in method call to 'SelectMemberByPathSelectionRule.SetSelectedPath(string)' calls nested method 'MemberPath.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Selection\ExcludeMemberByPathSelectionRule.cs	29
Argument for parameter 'differenceWithinPrecision' in method call to 'NumericAssertionsExtensions.FailIfDifferenceOutsidePrecision<float>(bool, NumericAssertions<float>, float, float, float, string, object[])' calls nested method 'float.IsPositiveInfinity(float)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\NumericAssertionsExtensions.cs	768
Argument for parameter 'differenceWithinPrecision' in method call to 'NumericAssertionsExtensions.FailIfDifferenceOutsidePrecision<float>(bool, NumericAssertions<float>, float, float, float, string, object[])' calls nested method 'float.IsNegativeInfinity(float)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\NumericAssertionsExtensions.cs	772
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Numeric\ComparableTypeAssertions.cs	109
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<TExpectation>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Numeric\ComparableTypeAssertions.cs	112
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Regex.IsMatch(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	554
Argument for parameter 'differenceWithinPrecision' in method call to 'NumericAssertionsExtensions.FailIfDifferenceOutsidePrecision<double>(bool, NumericAssertions<double>, double, double, double, string, object[])' calls nested method 'double.IsPositiveInfinity(double)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\NumericAssertionsExtensions.cs	900
Argument for parameter 'differenceWithinPrecision' in method call to 'NumericAssertionsExtensions.FailIfDifferenceOutsidePrecision<double>(bool, NumericAssertions<double>, double, double, double, string, object[])' calls nested method 'double.IsNegativeInfinity(double)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\NumericAssertionsExtensions.cs	904
Argument for parameter 'value' in method call to 'DateTime.AddTicks(long)' calls nested method 'Math.Min(long, long)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeAssertions.cs	169
Argument for parameter 'value' in method call to 'DateTime.AddTicks(long)' calls nested method 'Math.Min(long, long)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeAssertions.cs	214
Argument for parameter 'validValues' in method call to 'DateOnlyAssertions<TAssertions>.BeOneOf(IEnumerable<DateOnly?>, string, params object[])' calls nested method 'Enumerable.Cast<DateOnly?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateOnlyAssertions.cs	469
Argument for parameter 'validValues' in method call to 'DateOnlyAssertions<TAssertions>.BeOneOf(IEnumerable<DateOnly?>, string, params object[])' calls nested method 'Enumerable.Cast<DateOnly?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateOnlyAssertions.cs	487
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<DateOnly?>(IEnumerable<DateOnly?>, DateOnly?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateOnlyAssertions.cs	506
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DictionaryEquivalencyStep.cs	25
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsDictionaryItem<object, IDictionary>(object)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DictionaryEquivalencyStep.cs	25
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'DateTimeOffset.EqualsExact(DateTimeOffset)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	179
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'DateTimeOffset.EqualsExact(DateTimeOffset)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	217
Argument for parameter 'validValues' in method call to 'DateTimeAssertions<TAssertions>.BeOneOf(IEnumerable<DateTime?>, string, params object[])' calls nested method 'Enumerable.Cast<DateTime?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeAssertions.cs	853
Argument for parameter 'validValues' in method call to 'DateTimeAssertions<TAssertions>.BeOneOf(IEnumerable<DateTime?>, string, params object[])' calls nested method 'Enumerable.Cast<DateTime?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeAssertions.cs	871
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<DateTime?>(IEnumerable<DateTime?>, DateTime?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeAssertions.cs	890
Argument for parameter 'ticks' in method call to 'DateTimeOffset.AddTicks(long)' calls nested method 'Math.Min(long, long)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	305
Argument for parameter 'ticks' in method call to 'DateTimeOffset.AddTicks(long)' calls nested method 'Math.Min(long, long)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	350
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TimeSpanPredicate.IsMatchedBy(TimeSpan, TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetRangeAssertions.cs	80
Argument for parameter 'task2' in method call to 'Task.WhenAny(Task, Task)' calls nested method 'IClock.DelayAsync(TimeSpan, CancellationToken)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\AsyncFunctionAssertions.cs	318
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TimeSpanPredicate.IsMatchedBy(TimeSpan, TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeRangeAssertions.cs	81
Argument for parameter 'matchingRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<TSelf>.AddMatchingRule(IMemberMatchingRule)' calls nested method 'MustMatchByNameRule.MustMatchByNameRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	69
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'ByteArrayOrderingRule.ByteArrayOrderingRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	71
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataTableEquivalencyStep.cs	179
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TimeSpanPredicate.IsMatchedBy(TimeSpan, TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeRangeAssertions.cs	120
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<string>(IEnumerable<string>, string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\AssemblyAssertions.cs	94
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Nullable.Equals<TEnum>(TEnum?, TEnum?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\EnumAssertions.cs	80
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enum.IsDefined(Type, object)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\EnumAssertions.cs	150
Argument for parameter 'comparands' in method call to 'DataSetEquivalencyStep.CompareExtendedProperties(Comparands, IEquivalencyValidationContext, IEquivalencyValidator, IEquivalencyAssertionOptions, Dictionary<string, IMember>)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataSetEquivalencyStep.cs	130
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'HttpResponseMessageAssertions<TAssertions>.IsClientError()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\HttpResponseMessageAssertions.cs	136
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'HttpResponseMessageAssertions<TAssertions>.IsServerError()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\HttpResponseMessageAssertions.cs	164
Argument for parameter 'selectionRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<TSelf>.AddSelectionRule(IMemberSelectionRule)' calls nested method 'ExcludeMemberByPredicateSelectionRule.ExcludeMemberByPredicateSelectionRule(Expression<Func<IMemberInfo, bool>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	375
Argument for parameter 'selectionRule' in method call to 'SelfReferenceEquivalencyAssertionOptions<TSelf>.AddSelectionRule(IMemberSelectionRule)' calls nested method 'IncludeMemberByPredicateSelectionRule.IncludeMemberByPredicateSelectionRule(Expression<Func<IMemberInfo, bool>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	387
Argument for parameter 'item' in method call to 'List<IMemberMatchingRule>.Add(IMemberMatchingRule)' calls nested method 'TryMatchByNameRule.TryMatchByNameRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	398
Argument for parameter 'validValues' in method call to 'DateTimeOffsetAssertions<TAssertions>.BeOneOf(IEnumerable<DateTimeOffset?>, string, params object[])' calls nested method 'Enumerable.Cast<DateTimeOffset?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	1051
Argument for parameter 'item' in method call to 'List<IMemberMatchingRule>.Add(IMemberMatchingRule)' calls nested method 'MustMatchByNameRule.MustMatchByNameRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	408
Argument for parameter 'validValues' in method call to 'DateTimeOffsetAssertions<TAssertions>.BeOneOf(IEnumerable<DateTimeOffset?>, string, params object[])' calls nested method 'Enumerable.Cast<DateTimeOffset?>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	1069
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'MatchAllOrderingRule.MatchAllOrderingRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	543
Argument for parameter 'item' in method call to 'List<IEquivalencyStep>.Insert(int, IEquivalencyStep)' calls nested method 'EqualityComparerEquivalencyStep<T>.EqualityComparerEquivalencyStep(IEqualityComparer<T>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	532
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'PredicateBasedOrderingRule.PredicateBasedOrderingRule(Expression<Func<IObjectInfo, bool>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	553
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'ByteArrayOrderingRule.ByteArrayOrderingRule()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	563
Argument for parameter 'rule' in method call to 'OrderingRuleCollection.Add(IOrderingRule)' calls nested method 'PredicateBasedOrderingRule.PredicateBasedOrderingRule(Expression<Func<IObjectInfo, bool>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	573
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Condition(TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExecutionTimeAssertions.cs	77
Argument for parameter 'matchedConstraint' in method call to 'AndWhichConstraint<TAssertions, TEnum>.AndWhichConstraint(TAssertions, TEnum)' calls nested method 'TEnum?.GetValueOrDefault()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\NullableEnumAssertions.cs	47
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Condition(TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExecutionTimeAssertions.cs	110
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Condition(TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExecutionTimeAssertions.cs	140
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Condition(TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExecutionTimeAssertions.cs	173
Argument for parameter 'item' in method call to 'List<IEquivalencyStep>.Insert(int, IEquivalencyStep)' calls nested method 'AssertionRuleEquivalencyStep<TMember>.AssertionRuleEquivalencyStep(Expression<Func<IObjectInfo, bool>>, Action<IAssertionContext<TMember>>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	852
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TException>(IEnumerable<TException>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	87
Argument for parameter 'messages' in method call to 'ExceptionAssertions<TException>.ExceptionMessageAssertion.Execute(IEnumerable<string>, string, string, params object[])' calls nested method 'Enumerable.ToArray<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	90
Argument for parameter 'exceptions' in method call to 'ExceptionAssertions<TInnerException>.ExceptionAssertions(IEnumerable<TInnerException>)' calls nested method 'Enumerable.Cast<TInnerException>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	112
Argument for parameter 'exceptions' in method call to 'ExceptionAssertions<Exception>.ExceptionAssertions(IEnumerable<Exception>)' calls nested method 'ExceptionAssertions<TException>.AssertInnerExceptions(Type, string, params object[])'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	128
Argument for parameter 'exceptions' in method call to 'ExceptionAssertions<TInnerException>.ExceptionAssertions(IEnumerable<TInnerException>)' calls nested method 'Enumerable.Cast<TInnerException>(IEnumerable)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	147
Argument for parameter 'exceptions' in method call to 'ExceptionAssertions<Exception>.ExceptionAssertions(IEnumerable<Exception>)' calls nested method 'ExceptionAssertions<TException>.AssertInnerExceptionExactly(Type, string, params object[])'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	163
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	793
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	798
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	803
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'object.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	808
Argument for parameter 'value' in method call to 'StringBuilder.AppendLine(string?)' calls nested method 'ConversionSelector.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\SelfReferenceEquivalencyAssertionOptions.cs	811
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<TException, bool>.Invoke(TException)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	186
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<Exception>(IEnumerable<Exception>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	206
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<TSubject, TSubject, bool>.Invoke(TSubject, TSubject)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ObjectAssertions.cs	46
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'object.ReferenceEquals(object?, object?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ReferenceTypeAssertions.cs	85
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TException>(IEnumerable<TException>, Func<TException, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	224
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<Exception>(IEnumerable<Exception>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	235
Argument for parameter 'arg' in method call to 'Func<EquivalencyAssertionOptions<TExpectation>, EquivalencyAssertionOptions<TExpectation>>.Invoke(EquivalencyAssertionOptions<TExpectation>)' calls nested method 'AssertionOptions.CloneDefaults<TExpectation>()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ObjectAssertions.cs	129
Argument for parameter 'root' in method call to 'EquivalencyValidationContext.EquivalencyValidationContext(INode, IEquivalencyAssertionOptions)' calls nested method 'Node.From<TExpectation>(GetSubjectId)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ObjectAssertions.cs	131
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<TException, string>(IEnumerable<TException>, Func<TException, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	260
Argument for parameter 'failures' in method call to 'AssertionResultSet.AddSet(object, string[])' calls nested method 'AssertionScope.Discard()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\ExceptionAssertions.cs	287
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TException>(IEnumerable<TException>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Specialized\DelegateAssertionsBase.cs	41
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'string.EndsWith(string, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	760
Argument for parameter 'arg' in method call to 'Func<IObjectInfo, bool>.Invoke(IObjectInfo)' calls nested method 'ObjectInfo.ObjectInfo(Comparands, INode)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\AssertionRuleEquivalencyStep.cs	61
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'string.EndsWith(string, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	835
Argument for parameter 'obj' in method call to 'Action<IAssertionContext<TSubject>>.Invoke(IAssertionContext<TSubject>)' calls nested method 'AssertionContext<TSubject>.CreateFrom(Comparands, IEquivalencyValidationContext)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\AssertionRuleEquivalencyStep.cs	88
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'StringAssertions<TAssertions>.Contains(string, string, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	905
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'StringAssertions<TAssertions>.Contains(string, string, StringComparison)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	974
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Dictionary<string, Constraint>.TryGetValue(string, out Constraint)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintCollectionEquivalencyStep.cs	30
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Dictionary<string, Constraint>.TryGetValue(string, out Constraint)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintCollectionEquivalencyStep.cs	34
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.All<string>(IEnumerable<string>, Func<string, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	1041
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<string>(IEnumerable<string>, Func<string, bool>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	1077
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<DateTimeOffset?>(IEnumerable<DateTimeOffset?>, DateTimeOffset?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetAssertions.cs	1088
Argument for parameter 'types' in method call to 'TypeSelector.TypeSelector(IEnumerable<Type>)' calls nested method 'Assembly.GetTypes()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\TypeExtensions.cs	22
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	95
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<TEnum?, bool>.Invoke(TEnum?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\EnumAssertions.cs	385
Argument for parameter 'types' in method call to 'MethodInfoSelector.MethodInfoSelector(IEnumerable<Type>)' calls nested method 'Enumerable.ToList<Type>(IEnumerable<Type>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\TypeExtensions.cs	58
Argument for parameter 'types' in method call to 'PropertyInfoSelector.PropertyInfoSelector(IEnumerable<Type>)' calls nested method 'Enumerable.ToList<Type>(IEnumerable<Type>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\TypeExtensions.cs	78
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRowCollectionEquivalencyStep.cs	68
Argument for parameter 'propertyInfo' in method call to 'Property.Property(PropertyInfo, INode)' calls nested method 'Type.GetProperty(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	109
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	119
Argument for parameter 'condition' in method call to 'IAssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Contains<TEnum>(IEnumerable<TEnum>, TEnum)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\EnumAssertions.cs	425
Argument for parameter 'propertyInfo' in method call to 'Property.Property(PropertyInfo, INode)' calls nested method 'Type.GetProperty(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	145
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'string.IsNullOrEmpty(string?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	1337
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	155
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'string.IsNullOrWhiteSpace(string?)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\StringAssertions.cs	1377
Argument for parameter 'index' in method call to 'IEquivalencyValidationContext.AsCollectionItem<DataRow>(string)' calls nested method 'DataRowCollectionEquivalencyStep.CompoundKey.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRowCollectionEquivalencyStep.cs	161
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRowCollectionEquivalencyStep.cs	162
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	276
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRowEquivalencyStep.cs	175
Argument for parameter 'collection' in method call to 'HashSet<string>.HashSet(IEnumerable<string>)' calls nested method 'Enumerable.Select<DataColumn, string>(IEnumerable<DataColumn>, Func<DataColumn, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	203
Argument for parameter 'collection' in method call to 'HashSet<string>.HashSet(IEnumerable<string>)' calls nested method 'Enumerable.Select<DataColumn, string>(IEnumerable<DataColumn>, Func<DataColumn, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	204
Argument for parameter 'value' in method call to 'StringBuilder.Append(string?)' calls nested method 'Enumerable.Single<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	217
Argument for parameter 'value' in method call to 'StringBuilder.Append(string?)' calls nested method 'EnumerableExtensions.JoinUsingWritingStyle<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	221
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRowEquivalencyStep.cs	202
Argument for parameter 'value' in method call to 'StringBuilder.Append(string?)' calls nested method 'Enumerable.Single<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	236
Argument for parameter 'value' in method call to 'StringBuilder.Append(string?)' calls nested method 'EnumerableExtensions.JoinUsingWritingStyle<string>(IEnumerable<string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	240
Argument for parameter 'message' in method call to 'AssertionScope.FailWith(string)' calls nested method 'StringBuilder.ToString()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\ConstraintEquivalencyStep.cs	250
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'TimeSpanPredicate.IsMatchedBy(TimeSpan, TimeSpan)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\DateTimeOffsetRangeAssertions.cs	117
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Enumerable.Any<TAttribute>(IEnumerable<TAttribute>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\MemberInfoAssertions.cs	96
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'MethodInfoExtensions.IsNonVirtual(MethodInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\MethodInfoAssertions.cs	68
Argument for parameter 'values' in method call to 'string.Join(string?, IEnumerable<string?>)' calls nested method 'Enumerable.Select<Type, string>(IEnumerable<Type>, Func<Type, string>)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\MethodBaseAssertions.cs	100
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'MethodInfoExtensions.IsAsync(MethodInfo)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Types\MethodInfoAssertions.cs	96
Argument for parameter 'comparands' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'Comparands.Comparands(object, object, Type)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataSetEquivalencyStep.cs	220
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	101
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataSetEquivalencyStep.cs	244
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataColumnEquivalencyStep.cs	91
Argument for parameter 'compareTable' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	113
Argument for parameter 'compareColumns' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	114
Argument for parameter 'compareKeyConstraint' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	115
Argument for parameter 'compareTable' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	122
Argument for parameter 'compareColumns' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	123
Argument for parameter 'compareKeyConstraint' in method call to 'DataRelationEquivalencyStep.CompareDataRelationConstraints(IEquivalencyValidator, IEquivalencyValidationContext, DataRelation, DataRelation, Dictionary<string, IMember>, string, bool, bool, bool, Func<DataRelation, DataColumn[]>, Func<DataRelation, DataTable>)' calls nested method 'Dictionary<string, IMember>.ContainsKey(string)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	124
Argument for parameter 'context' in method call to 'IEquivalencyValidator.RecursivelyAssertEquality(Comparands, IEquivalencyValidationContext)' calls nested method 'IEquivalencyValidationContext.AsNestedMember(IMember)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	216
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataColumnEquivalencyStep.cs	139
Argument for parameter 'context' in method call to 'IMemberSelectionRule.SelectMembers(INode, IEnumerable<IMember>, MemberSelectionContext)' calls nested method 'MemberSelectionContext.MemberSelectionContext(Type, Type, IEquivalencyAssertionOptions)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Equivalency\Steps\DataRelationEquivalencyStep.cs	240
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ReferenceTypeAssertions.cs	299
Argument for parameter 'condition' in method call to 'AssertionScope.ForCondition(bool)' calls nested method 'Func<T, bool>.Invoke(T)'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ReferenceTypeAssertions.cs	409
Argument for parameter 'c' in method call to 'Type.IsAssignableFrom(Type?)' calls nested method 'object.GetType()'	FluentAssertions	C:\dev\fluentassertions\Src\FluentAssertions\Primitives\ReferenceTypeAssertions.cs	357
```

</details>